docs/cli-help.md

```diff
@@ -1,6 +1,6 @@
 # Command-line Help
 
-This document describes the commands available in the `gateway` binary of `nginx-kubernetes-gateway` container.
+This document describes the commands available in the `gateway` binary of the `nginx-gateway` container.
```

Comment:
```suggestion
This document describes the commands available in the `gateway` binary of the `nginx-gateway-fabric` container.
````

Reply:
The name of the container is still `nginx-gateway`.